loops.js

```diff
@@ -51,3 +65,13 @@
 // >> 9
 // ==========================================
 
+sign = "";
+
+for (let i = 0; i < 10; i++) {
+    if (i >= 3 && i < 6) {
+        sign = "> ";
+    } else if (i >= 6) {
+        sign = ">> ";
+    }
+    console.log(sign + i);
+}
```

Comment:
Ooit eens gelezen dat het veiliger is om elk bestand te laten eindigen met een lege regel.
Iets met dat bestanden anders door kunnen lopen in het volgende bestand in de compiler..ook als dat niet de bedoeling is..maar begreep zelf niet helemaal waarom.
Even gezocht naar wat uitleg: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
NB: Github geeft ook een rood meldinkje op deze plek ivm het ontbreken van de lege regel.